connect/transforms/src/main/java/org/apache/kafka/connect/transforms/util/Requirements.java

```diff
@@ -32,6 +32,7 @@ public static void requireSchema(Schema schema, String purpose) {
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Since you are touching this class, I noticed use of `getCanonicalName`. Do we want to change to `getName` here too? It doesn't really matter, it's only used for logging, but I mention it for consistency.